README.md

```diff
@@ -1,3 +1,16 @@
 # Student Register
 
 Tool to register a list of users in the form of `users.csv` with DCU SU
+
+## Setup:
+```
+npm install
+cp .sample.env .env
+vim .env
```

Comment:
telling people yo use vim isnt the best

Reply:
Meh I know but most people who have an opinion will just change it out anyway